src/components/AppList/index.js

```diff
@@ -0,0 +1,159 @@
+import i18n from '@dhis2/d2-i18n'
+import { PropTypes } from '@dhis2/prop-types'
+import {
+    NoticeBox,
+    CenteredContent,
+    CircularLoader,
+    InputField,
+} from '@dhis2/ui'
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+import { useQueryParam, StringParam, withDefault } from 'use-query-params'
+import AppIcon from './AppIcon'
+import styles from './AppList.module.css'
+
+const AppCards = ({ apps }) => {
+    const history = useHistory()
+    const createClickHandler = appHubId => () => {
+        if (appHubId) {
+            history.push(`/app/${appHubId}`)
+        }
+    }
+
+    return (
+        <div className={styles.appCards}>
+            {apps.map(app => (
+                <button
+                    key={app.short_name}
+                    className={styles.appCard}
+                    disabled={!app.appHub?.id}
+                    onClick={createClickHandler(app.appHub?.id)}
```

Comment:
(minor, could be fixed later) - we do want to show details about installed apps even if they don't have an appHub ID - version, developer, etc (things from the manifest) still make sense here.